coreneuron/io/reports/report_event.cpp

```diff
@@ -24,12 +24,14 @@ ReportEvent::ReportEvent(double dt,
                          double tstart,
                          const VarsToReport& filtered_gids,
                          const char* name,
-                         double report_dt)
+                         double report_dt,
+                         int report_index)
     : dt(dt)
     , tstart(tstart)
     , report_path(name)
     , report_dt(report_dt)
-    , vars_to_report(filtered_gids) {
+    , vars_to_report(filtered_gids)
+    , report_t_shift_(1e-6 * report_index) {
```

Comment:
just to avoid hardcoded factor - what if we replace `1e-6` with  `dt/1e-3`? i.e. whatever will be the value of dt in the future, we can always have this factor 1e3 times lower.